packages/cli/src/actions/add.ts

```diff
@@ -0,0 +1,24 @@
+import { PIPCOOK_BOA } from '../constants';
+import { execAsync, logger } from '../utils/common';
+
+const supportDeps = [ 'tensorflow', 'tvm' ];
+
+/**
+ * Add missing deps
+ */
+const add = async () => {
+  if (process.argv.length !== 4) { return; }
+  const name = process.argv[3];
+  if (supportDeps.includes(name)) {
+    if (name === 'tvm') {
+      logger.info('Installing tvm ...');
```

Comment:
This might be unnecessary because the below command would also print the related installation log.